frontend/src/components/ui/chat/ChatTextBox.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from "react";
+
+import style from "./ChatTextBox.module.css";
+
+function ChatTextBox() {
+  const [state] = useState({ message: "" });
```

Comment:
Whoa didn't know it works this way.

What I usually do is `const <state> = useState(<initial_state>)[0]` if I do not need the <setState> function since the index [0] shows that clearly. But I think this way is cleaner since not using magic number 0, good job!

Reply:
Thanks fam!